sources/packages/backend/apps/api/src/route-controllers/overaward/models/overaward.dto.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  AssessmentTriggerType,
+  DisbursementOverawardOriginType,
+} from "@sims/sims-db";
+import {
+  AWARD_VALUE_CODE_LENGTH,
+  MAXIMUM_AWARD_VALUE,
+} from "../../../utilities";
+import { Length, Max, Min } from "class-validator";
+
+export class OverawardBalanceAPIOutDTO {
+  overawardBalanceValues: Record<string, number>;
+}
+
+export class OverawardAPIOutDTO {
+  dateAdded: Date;
+  overawardOrigin: DisbursementOverawardOriginType;
+  awardValueCode: string;
+  overawardValue: number;
+  addedByUserFirstName?: string;
```

Comment:
Are not usually returning from the API the user name as formatted to be directly used using the method `getUserFullName`?